### DISCUSSION TITLE
For fields like sector, product, location type, what are the existing standards we should reference?

### DISCUSSION THREAD

**Discussion by shuyag** [opensupplyhub/supplychaindata.exchange#18] [General]
Starting this issue to collect existing standards to reference for these fields all in one place! Pulling in the thread that @vasgat and @fredsourcing started in Slack here: 

Organization types:
- [SIC] (https://www.sec.gov/search-filings/standard-industrial-classification-sic-code-list)

Sector types:
- [ISIC (International Standard Industrial Classification)](https://unstats.un.org/unsd/trade/events/2013/foc/recommendations/1b%20-%20UN%20ISIC,%20Revision%204.pdf)
- [NACE (Nomenclature of Economic Activities)] (https://ec.europa.eu/eurostat/documents/3859598/5902521/KS-RA-07-015-EN.PDF)

Product types:
- [GPC - GS1 Global Product Classification](https://gpc-browser.gs1.org/)
- [United Nations Standard Products and Services Code (UNSPSC)](https://store.unspsc.org/collections/codeset-downloads?_gl=1*1q0a9yd*_gcl_au*MTY5Mzc4MDE4NC4xNzI1ODk1OTk2) or [here](https://www.commerce.gov/oam/resources/united-nations-standard-products-and-services-codes-unspsc) on the [commerce.gov](http://commerce.gov/)

**Comment by fredruss** [comment]
We could standardise countries too - I suggest either Alpha-2 or Alpha-3 codes.

**Comment by shuyag** [comment]
Good addition, OSIDs are using ISO 3166 for country codes too, dropping [here for reference](https://www.iban.com/country-codes). Theoretically could use numeric as well, but the alpha-2/alpha-3 would be more readable.
